db/seeds.rb

```diff
@@ -115,7 +115,7 @@
 submission = Submission.create!( solution: solution, uuid: SecureRandom.uuid, submitted_via: "cli")
 submission.files.create!( filename: "lasagna.rb", content: "class Lasagna\nend", digest: SecureRandom.uuid)
 Iteration.create!( submission: submission, solution: solution, idx: 1)
-Solution::MentorRequest.create!(solution: solution)
+Solution::MentorRequest.create!(solution: solution, comment_markdown: "I would like to improve the performance of my code")
```

Comment:
Without this addition this results in a validation error.

Reply:
Nicely spotted :)